list.json

```diff
@@ -0,0 +1,43 @@
+{
+    "1": {
+        "status": "[x]",
+        "name": "Bikin Aplikasi",
+        "tag": [],
+        "createDate": "2020-04-07T18:02:49.741Z",
+        "completeDate": "2020-04-07T18:02:49.741Z"
+    },
+    "2": {
```

Comment:
seharusnya bentuknya cukup arr of object saja
[
{
id : 1
status : "x"
tag: []
dll
},
{
id : 2
status : "[ ]"
tag: []
dll
},
]